models/seed-models.js

```diff
@@ -49,20 +49,47 @@ exports.fetchUsers = () => {
   });
 };
 
-exports.fetchArticles = () => {
-  return db
-    .query(
-      `
+exports.fetchArticles = (sort_by = 'created_at', order_by = 'DESC', topic) => {
+   
+    const validSortBys = ['article_id', 'title', 'body', 'author', 'topic', 'created_at', 'votes', 'comment_count']
+    const validOrderBys = ['ASC', 'DESC']
+    const validTopics = ['mitch', 'cats']
```

Comment:
I would say this will be very tricky to green-list as any new topics added will need to be manually included in this list. This would be better suited to another `checkIfExists` pattern to determine exactly why the array is empty that comes from the database